integration/kubernetes/singleMaster-hdfsJournal/config.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+#
+# This script generates a Docker image for the repository from its current state.
+#
+
+resources:
+  masterCount: 1
+
+journal:
+  type: "UFS"
+  ufsType: "HDFS"
```

Comment:
can we detect this from folder path by searching for prefix instead? maybe address this in a different PR

Reply:
I'll address this in a different PR